### PR TITLE
doc PushTester by fixing javadoc solving #52

### LINKED ISSUE #52: doc fix JavaDoc in PushTester
- [ ] add a JavaDoc for PushTester
- [ ] wrong parameter names are used in all functions in PushTeser and return wrong parameters (see image below).   

image

### PR DESCRIPTION
**Pull request by juliaBystrom**
If accepted will close #52

### CODE REVIEW

File: ci/src/main/java/ci/PushTester.java [hunk 1]
```diff
@@ -33,9 +39,9 @@ public PushStatus createPushStatus(File dir, String commitID, String commitDate)
 
 
     /**
-     * Build the directory inputed using gradle
+     * Build and test the directory inputed using gradle. Sets the class variables depending on the result.
      *
-     * @param Local folder
+     * @param dir File  A file object for the github repository.
      * @return void
```
Comment (ElisabethChen) [inline ci/src/main/java/ci/PushTester.java:45]: Does it need a return when it return void?

**Review by ElisabethChen — APPROVED** [review]
Looks good to me! added one commet that is a question but the content looks great!

File: ci/src/main/java/ci/PushTester.java (hunk 1 above)
Comment (cadye) [reply ci/src/main/java/ci/PushTester.java:45]: I assume you always have a return, no matter what it is. no idea though

**Review by cadye — APPROVED** [review]
some grammatical errors and typos, but overall ok.

**Review by hemenali11 — APPROVED** [review]

**Pull request merged**